contrib/tidwall/buntdb/buntdb.go

```diff
@@ -93,7 +93,7 @@ func WrapTx(tx *buntdb.Tx, opts ...Option) *Tx {
 	}
 }
 
-func (tx *Tx) startSpan(name string) ddtrace.Span {
+func (tx *Tx) startSpan(name string) ddtrace.SpanW3C {
```

Comment:
We should not need to update these contrib packages (except where we want to use the new ID).

If we have to, that means we're breaking our API.